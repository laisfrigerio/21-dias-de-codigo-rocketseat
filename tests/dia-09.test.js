const { JSDOM } = require('jsdom')
const fs = require('fs')
const path = require('path')

const html = fs.readFileSync(path.resolve(__dirname, '../dia-09/index.html'), 'utf8')

let dom
let container

describe('dia 09', () => {
  beforeEach(() => {
    dom = new JSDOM(html, { runScripts: 'dangerously' })
    container = dom.window.document.body
  })

  test('should match the loading challenge', () => {
    expect(container).toMatchSnapshot();
  });
})