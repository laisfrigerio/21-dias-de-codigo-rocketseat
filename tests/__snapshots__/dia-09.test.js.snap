// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dia 09 should match the loading challenge 1`] = `
<body
  class="main-container no-toggle"
  id="particles-js"
>
  
  
  <header />
  
  
  <div
    class="spinner"
  />
  
  
  <script
    src="../assets/js/particles.js"
  />
  
  
  <script
    src="../assets/js/include-html.js"
  />
  
  
  <script
    src="scripts.js"
  />
  
  
  <script
    src="https://cdn.jsdelivr.net/npm/particles.js@2.0.0/particles.min.js"
  />
  



</body>
`;
